### DISCUSSION TITLE
<service>|Styles|Web map using the style

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#1168] [Configuration]
**"Web map using the style**": Is it possible to limit the extent to the spatial extent of the data?
![Screenshot 2024-02-14 122134](https://github.com/interactive-instruments/ldproxy/assets/1905113/795c9467-c864-4a5c-aa2d-fbb9e43b7188)

**Comment by cportele** [comment]
The map will use the `center`, `zoom`, `pitch`, `bearing` information in the style as specified in the [MapLibre Style spec](https://maplibre.org/maplibre-style-spec/root/).
